db/schema.rb

```diff
@@ -182,6 +182,7 @@
     t.string "webpage"
     t.string "message"
     t.boolean "send_digest", default: false
+    t.string "threshold_hash"
```

Comment:
This might be better named as something along the lines of current_condition_threshold_hash  -- or something shorter that at least references that this is not the threshold_hash of the jurisdiction but rather for the jurisdictions most up to date threshold condition